frontend/packages/policy-editor/src/components/ConsentResourcePolicyRulesEditor/ConsentResourcePolicyRulesEditor.tsx

```diff
@@ -0,0 +1,145 @@
+import { PolicyRuleContextProvider } from '../../contexts/PolicyRuleContext';
+import type { PolicyError, PolicyRuleCard } from '../../types';
+import React, { useState, useId } from 'react';
+import { PolicySubjects } from '../PolicyCardRules/PolicyRule/PolicySubjects';
+import { PolicyAccessPackages } from '../PolicyCardRules/PolicyRule/PolicyAccessPackages';
+import { PolicyRuleErrorMessage } from '../PolicyCardRules/PolicyRule/PolicyRuleErrorMessage';
+import { usePolicyEditorContext } from '../../contexts/PolicyEditorContext';
+import { accessListSubjectSource, organizationSubject } from '../../utils';
+import {
+  StudioCheckbox,
+  StudioErrorMessage,
+  StudioHeading,
+  StudioFieldset,
+} from '@studio/components-legacy';
```

Comment:
We are transitioning to the new `@studio/components` package, and all these components exists in the new library and should therefore be used instead 😄